whetstone/runtime/src/main/java/com/freeletics/mad/whetstone/internal/NavEntryComponentGetter.kt

```diff
@@ -0,0 +1,37 @@
+package com.freeletics.mad.whetstone.internal
+
+import android.content.Context
+import androidx.navigation.NavBackStackEntry
+import com.freeletics.mad.whetstone.NavEntryComponent
+import com.freeletics.mad.whetstone.NavEntryId
+import dagger.MapKey
+import kotlin.reflect.KClass
+
+/**
+ * A generated implementation of this can be used to retrieve a generated [NavEntryComponent].
+ *
+ * The implementation will be bound into a `Map<String, NavEntryComponentGetter>` were the key
+ * is the same scope that is used in [NavEntryId] and [NavEntryComponent].
```

Comment:
```suggestion
 * The implementation will be bound into a `Map<Class<*>, NavEntryComponentGetter>` were the key
 * is the same scope that is used in [NavEntryId] and [NavEntryComponent]. It can be used through
 * [com.freeletics.mad.whetstone.NavEntryComponents].
```